pkg/controller/sparkapplication/controller.go

```diff
@@ -972,18 +977,18 @@ func (c *Controller) recordDriverEvent(app *v1beta2.SparkApplication, phase v1be
 	}
 }
 
-func (c *Controller) recordExecutorEvent(app *v1beta2.SparkApplication, state v1beta2.ExecutorState, name string) {
+func (c *Controller) recordExecutorEvent(app *v1beta2.SparkApplication, state v1beta2.ExecutorState, args ...interface{}) {
 	switch state {
 	case v1beta2.ExecutorCompletedState:
-		c.recorder.Eventf(app, apiv1.EventTypeNormal, "SparkExecutorCompleted", "Executor %s completed", name)
```

Comment:
How does the output look like for `args`?

Reply:
Same as before we already use `c.recorder.Eventf` which already accept variadics parameters

per doc:
```
Eventf(object runtime.Object, eventtype, reason, messageFmt string, args ...interface{})
```

We already use something similar in `recordSparkApplicationEvent` for `v1beta2.FailedState`